documentation/modules/proc-deploying-api-server.adoc

```diff
@@ -34,10 +28,10 @@ sed -i 's/myproject/_<my-namespace>_/' install/components/api-server/*RoleBindin
 ifeval::["{cmdcli}" == "kubectl"]
 . Create a certificate to use with the API server. For testing purposes, you can create a self-signed certificate:
 +
-[options="nowrap"]
+[options="nowrap",subs="+quotes,attributes"]
 ----
 mkdir -p api-server-cert/
-openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=api-server.enmasse.svc.cluster.local" -out api-server-cert/tls.crt -keyout api-server-cert/tls.key
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=api-server.{ProductNamespace}.svc.cluster.local" -out api-server-cert/tls.crt -keyout api-server-cert/tls.key
 ----
```

Comment:
<{ProductNamespace}>

Add:
"where <{ProductNamespace}> is the namespace where {ProductName} is deployed."